app/views/contact/govuk/accessible_format_requests/start_page.html.erb

```diff
@@ -0,0 +1,50 @@
+<% content_for :title do %>
+  <% if flash[:input_errors]&.any? %>
```

Comment:
Will flash work here, and... is it necessary? There aren't going to be any input errors on this page, are there? There's no input.

Reply:
Thanks Keith, it's not necessary and has been removed.